specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/computeRPCommon.json

```diff
@@ -1798,9 +1802,49 @@
             "name": "LinuxPatchAssessmentMode",
             "modelAsString": true
           }
+        },
+        "automaticByPlatformSettings": {
+          "$ref": "#/definitions/LinuxAutomaticByPatchSettings",
+          "description": "Specifies additional settings for patch mode AutomaticByPlatform in VM Guest Patching on Linux."
         }
       },
       "description": "Specifies settings related to VM Guest Patching on Linux."
+    },
+    "WindowsAutomaticByPatchSettings": {
+      "properties": {
+        "rebootSetting": {
+          "type": "string",
+          "description": "Specifies reboot setting for all AutomaticByPlatform patch installation operations.",
```

Comment:
"Specifies **the** reboot setting for all AutomaticByPlatform patch installation operations."

For Linux as well.

Reply:
Resolved